simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFeedFragment.kt

```
@@ -20,108 +20,106 @@ import android.widget.RadioButton
 import org.nypl.simplified.feeds.api.FeedGroup
-import org.nypl.simplified.feeds.api.FeedLoaderResult.FeedLoaderFailure
-import org.nypl.simplified.feeds.api.FeedLoaderResult.FeedLoaderFailure.FeedLoaderFailedAuthentication
-import org.nypl.simplified.feeds.api.FeedLoaderResult.FeedLoaderFailure.FeedLoaderFailedGeneral
-import org.nypl.simplified.feeds.api.FeedLoaderType
 import org.nypl.simplified.feeds.api.FeedSearch
 import org.nypl.simplified.listeners.api.FragmentListenerType
 import org.nypl.simplified.listeners.api.fragmentListeners
-import org.nypl.simplified.profiles.api.ProfileEvent
-import org.nypl.simplified.profiles.api.ProfileUpdated
-import org.nypl.simplified.profiles.controller.api.ProfilesControllerType
-import org.nypl.simplified.taskrecorder.api.TaskRecorder
 import org.nypl.simplified.ui.accounts.AccountPickerDialogFragment
-import org.nypl.simplified.ui.catalog.CatalogFeedArguments.CatalogFeedArgumentsRemote
 import org.nypl.simplified.ui.catalog.CatalogFeedOwnership.CollectedFromAccounts
 import org.nypl.simplified.ui.catalog.CatalogFeedOwnership.OwnedByAccount
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedAgeGate
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoadFailed
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoaded.CatalogFeedEmpty
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoaded.CatalogFeedNavigation
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoaded.CatalogFeedWithGroups
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoaded.CatalogFeedWithoutGroups
-import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoading
-import org.nypl.simplified.ui.errorpage.ErrorPageParameters
-import org.nypl.simplified.ui.images.ImageLoaderType
+import org.nypl.simplified.ui.catalog.CatalogFeedState.*
+import org.nypl.simplified.ui.catalog.CatalogFeedState.CatalogFeedLoaded.*
```

Star imports won't pass `ktlint`.

---

Why did they, both locally and on CI?

---

Great, seems like `ktlint` isn't being run at all.

The build script contains these lines:

```
subprojects {
  assemble.dependsOn ktlint
}
```

So, naturally, they just fail silently and `ktlint` never runs.

---

I'm going to OK this, because I'm going to submit a PR afterwards that makes `ktlint` run again. I can fix style issues then.